plugins/modules/zos_job_output.py

```diff
@@ -195,11 +195,34 @@
              Return code converted to integer value (when possible).
           type: int
           sample: 00
+        steps:
+          description:
+            Series of JCL steps that were executed and their return codes.
+          type: list
+          elements: dict
+          contains:
+            step_name:
+              description:
+                Name of the step shown as "was executed" in the DD section.
+              type: str
+              sample: "STEP0001"
+            step_cc:
+              description:
+                Actual CC returned for this step in the DD section.
```

Comment:
Real minor, using "Actual" might imply that elsewhere in the code we are deducing RC's. I would  replace "Actual" with "The" so it reads "The condition code (cc) returned for......"

Reply:
Hadn't thought about that.  Done